### DISCUSSION TITLE
npm bootbox dist different from the master branch in github

### DISCUSSION THREAD

**Discussion by francisloumendoza** [makeusabrew/bootbox#865] [General]
I compared the bootbox.js from the npm vs. what's on the master branch and they are different. I'm specifically looking at the changes on the deprecated jQuery. (e.g. on npm it is still using the deprecated codes "$.isFunction(options.onHide)" while on github master, its now changed to "typeof options.onHide === 'function'" )

On the notes on both files, it says 

 /*! @preserve
 * bootbox.js
 * version: 6.0.0
 * author: Nick Payne <nick@kurai.co.uk>
 * license: MIT
 * http://bootboxjs.com/
 */

Shouldn't it be version 6.0.1 or something?

**Comment by tiesont** [comment]
> Shouldn't it be version 6.0.1 or something?

Probably. Been awhile since I pushed an update to npm. There's an issue (#862) that I want to address and then I'll push another release (no ETA, sorry). When that happens I'll try to make sure that npm gets updated as well.

**Comment by tiesont** [comment]
I've pushed package updates out for two versions, v6.0.1 and v6.0.2. Should be available shortly, if not already.
